draft-packages/form/docs/CheckboxGroup.stories.tsx

```diff
@@ -284,3 +284,69 @@ export const WithoutBottomMargin = () => (
 )
 
 WithoutBottomMargin.storyName = "without bottom margin"
+
+export const ConditionallyChecked = () => {
+  const [selectedOptions, setSelectedOptions] = React.useState<number[]>([])
+
+  const onCheckHandler = (state: string, value: number) => {
+    if (state === "off") {
```

Comment:
I'd consider calling the story `Checkbox nested values` or something like that since `ConditionallyChecked` is too broad.